src/wasm-interpreter.h

```diff
@@ -35,6 +35,24 @@ using namespace cashew;
 
 IString WASM("wasm");
 
+
+#ifdef WIN32
+#include <intrin.h>
+
+int32_t safe_clz(int32_t v) {
+  unsigned long result;
+  return _BitScanReverse(&result,v) ? result : 32;
+}
+
+int32_t safe_ctz(int32_t v) {
+  unsigned long result;
+  return _BitScanForward(&result,v) ? result : 32;
+}
+
+int32_t platform_popcount(int32_t v) {
+  return __popcnt(v);
```

Comment:
I can import the following: http://src.chromium.org/native_client/trunk/src/native_client/src/include/portability_bits.h

It would require me to do it since it involves re-licensing code Google has copyrighted, I can do it in a separate PR if that's OK with @kripken.